tests/test_surface/test_file_io.py

```diff
@@ -62,7 +62,7 @@ def surfaces(draw):
         "irap_binary",
         "irap_ascii",
         "zmap_ascii",
-        # "ijxyz",  # This fails horribly
+        "ijxyz",
```

Comment:
I am still a bit worried that this test can be flaky. the reason for that is that the `ijxyz` format may change ncol and/or nrow in cases with undefs along the border, which is not possible to avoid.